fe/src/main/java/org/apache/doris/analysis/ExpressionFunctions.java

```diff
@@ -162,13 +160,52 @@ public FEFunctionSignature getSignature() {
             return signature;
         }
 
+        // Now we only support functions which contain only one variable-length arg or fixed-length args 
+        // which does't contain any arrays.
         public LiteralExpr invoke(List<Expr> args) throws AnalysisException {
+            boolean isArray = false;
+            for (Class<?> argType : method.getParameterTypes()) {
+                if (argType.isArray()) {
+                    Preconditions.checkArgument(method.getParameterTypes().length == 1);
```

Comment:
Why using Preconditions, not throw an exception?

Reply:
because it should't appear and it's a programing error. But this code has been removed.